community/sharing.md

```diff
@@ -0,0 +1,90 @@
+# Sharing your Mods
+Author(s): code3z (@code3 on the forum)
+
+Once you've created a mod that you're proud of, the next step is to share it.
+Sharing will be a big help to other people. You can also get feedback and help
+from the community. But it may also mean you have to respond to bugs in your
+code and help other people use your code.
+
+## Managing and Sharing Mods using Git
+
+You can use hosted Git repos such as GitHub and Gitlab to upload, manage, share,
+and collaborate on mods. See the Managing Mods section of this guide for more
```

Comment:
*managing mods guide*

…and I don’t plan to explain much Git there, just mention it & maybe link a tutorial.

Reply:
Well, maybe mention briefly how to work with it - not actually what things to click in the GUI or commands to type in the CLI, just why you should use it and some modding-specific things about Git.